apps/pom.xml

```diff
@@ -1025,9 +1025,15 @@
                     <failOnWarnings>false</failOnWarnings>
                     <detectJavaApiLink>false</detectJavaApiLink>
                     <!-- Exclude javadocs on certain generated files, reduces build warnings -->
-                    <excludePackageNames>gov.cms.mpsm.rda.*</excludePackageNames>
-                    <excludePackageNames>gov.cms.bfd.model.rif.entities.*</excludePackageNames>
-                    <excludePackageNames>gov.cms.bfd.model.rda.entities.*</excludePackageNames>
+                    <excludePackageNames>
+                        gov.cms.bfd.pipeline.rda.grpc.*;
+                        gov.cms.mpsm.rda.v1.*;
+                        gov.cms.mpsm.rda.v1;
+                        gov.cms.bfd.model.rda;
+                        gov.cms.bfd.model.rda.*;
+                        gov.cms.bfd.model.rif;
+                        gov.cms.bfd.model.rif.*;
+                    </excludePackageNames>
```

Comment:
As we discussed this should be changed to use sourceFileExcludes as that also allows the JPA generated metamodel classes.